docker/debug-pod/Dockerfile

```
@@ -0,0 +1,62 @@
+    iproute2 \
+    telnet \
+    make \
+    gcc \
+    mysql-client \
+    postgresql-client \
+    jq \
+    python3 \
+    python3-pip
+
+# ETCD CLI Install
+RUN ARCH=$(dpkg --print-architecture) && \
```

Hadolint (2.12.0)

[error] Do not use sudo as it leads to unpredictable behavior. Use a tool like gosu to enforce root

(DL3004)